pkg/cloudprovider/providers/azure/azure_managedDiskController.go

```diff
@@ -174,7 +174,7 @@ func (c *ManagedDiskController) getDisk(resourceGroup, diskName string) (string,
 		return *(*result.DiskProperties).ProvisioningState, *result.ID, nil
 	}
 
-	return "", "", err
+	return "", "", fmt.Errorf("Invalid response for getDisk")
```

Comment:
maybe we should say what the result is to aid in troubleshooting?

Reply:
> maybe we should say what the result is to aid in troubleshooting?

so how to describe it 
such as "can not get disk provisionState and diskID "
or
"disk DiskProperties  is nil or provisionState is nil "
Thanks.

Reply:
I think we should just keep the original code, original `err` is nil in Line 177, just keep it.